src/Modules/BrainStimulator/SetupRHSforTDCSandTMS.cc

```diff
@@ -63,6 +63,8 @@ void SetupTDCS::setStateDefaults()
   setStateDoubleFromAlgo(Parameters::normal_dot_product_bound);
   setStateDoubleFromAlgo(Parameters::pointdistancebound);
   setStateIntFromAlgo(Parameters::number_of_electrodes);
+  setStateBoolFromAlgo(Parameters::GetContactSurface);
+  state->setValue(Parameters::GetContactSurface, false);
```

Comment:
Yes, I had problems with that one. The GetContactSurface boolean gets saved in the network file but
after reloading it does not show up as being "checked" in the GUI. Can you have a look at that?
Thanks!